charts/devices-api/values.yaml

```diff
@@ -120,6 +120,7 @@ env:
     -----END CERTIFICATE-----
   IPFS_URL: https://assets.dev.dimo.xyz/ipfs
   SD_INFO_TOPIC: table.task.synthetic.fill
+  PRIVACY_FENCE_TOPIC_V2: table.device.privacyfence.v2
```

Comment:
looks like you has another pr for table creation in kafka

Reply:
I was playing with too many branches. It's in here now.